src/cmd/api/docs/swagger.json

```diff
@@ -0,0 +1,222 @@
+{
```

Comment:
It will be awesome if we can find a way to check in the CI that the file generated with swagger is up to date.